Doc/library/asyncio-task.rst

```diff
@@ -225,6 +225,19 @@ Running an asyncio Program
     the end.  It should be used as a main entry point for asyncio
     programs, and should ideally only be called once.
 
+    Return the `base_events.run_until_complete()` that this return the
```

Comment:
I think we don't need to mention a future and `run_until_complete()` here.
`asyncio.run()` returns a result of *coro* execution, that's it.

Reply:
Hi @asvetlov Thanks for the review. I make the changes